src/components/router-link.tsx

```diff
@@ -0,0 +1,17 @@
+import {Link as MuiLink} from "@mui/material";
+import React from "react";
+import {Link as RouterLink} from "react-router-dom";
+
+export function Link({to, children}: {to: string; children: React.ReactNode}) {
```

Comment:
great idea on making these entry functions clickable. but i'm thinking instead of making the function name a deeplink, we instead could have a little icon next to it, and clicking the icon should lead us to the module page. reason why: we have this [design](https://www.figma.com/proto/gG4UP6OipyXq0LjM4zWmMw/Explorer-Design-Exploration?node-id=2-1602&starting-point-node-id=2%3A1602) for explorer improvement, where having a icon next to a field either to copy it or click it is gonna be more consistent.

Reply:
hmm, so you're saying clicking function name should do nothing?
I think it's still reasonable and better UX to have the function name itself clickable and perform a default action (i.e. in this case navigate to the function).
Also in the linked design I would suggest the same for the sender and sent to addresses - i.e. click on the address takes you to it, click on the copy icon next to it copies the full address.